packages/rules/src/consistentVersions.ts

```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
     });
   }
 };
+
+const ensurePackageMatchesSomeVersion = (
+  context: Context,
+  dependencyPackageName: string,
+  acceptedPackageDependencyValues: string[]
+) => {
+  const packageJson = context.getPackageJson();
+  const packageJsonPath = context.getPackageJsonPath();
+
+  const acceptedPackageDependencyVersions: SemVer[] = acceptedPackageDependencyValues.map(
+    acceptedPackageDependencyValue => {
+      const acceptedPackageDependencyVersion = coerce(acceptedPackageDependencyValue);
+      if (acceptedPackageDependencyVersion == null) {
+        throw new Error(
+          `Malformed accepted package dependency version defined in monorepolint configuration: ${dependencyPackageName} @ '${acceptedPackageDependencyValue}'`
+        );
+      }
+      return acceptedPackageDependencyVersion;
+    }
+  );
+
+  const actualPackageDependencyValue = packageJson.dependencies && packageJson.dependencies[dependencyPackageName];
+  const actualPackageDependencyVersion = coerce(actualPackageDependencyValue);
+  if (
+    actualPackageDependencyVersion != null &&
+    !acceptedPackageDependencyVersions.some(
```

Comment:
these suggestions are equivalent it's just how my brain wants to read this

```suggestion
    acceptedPackageDependencyVersions.every(
```

Reply:
hmmmmm, my brain thought of "you must match one of these" as `if ("you don't match one of these") => Error`.

So I thought `if (!"you match one of these" ) => Error`,
as opposed to `if ("you don't match any of these") => Error`

The English for the latter seems smoother, though the code reads to me as `if ("all of these you do not match") => Error`.
This has been fun.
Gonna change it 👍